src/interface/src/app/treatments/direct-impacts-maps-panel/direct-impacts-synced-maps.component.scss

```diff
@@ -0,0 +1,13 @@
+@import "colors";
+
+$panel-height: 1000px;
+
+:host {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  gap: 1px;
+  height: $panel-height;
```

Comment:
Just learning from your PR, why we are using fixed height instead of dynamic ones? Is there any limitation in the map library

Reply:
I made a comment on figma about this- not sure what drives the overall height yet. Let me add a comment